File: Python/A_NivelIniciante/Exercicio_1040.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1040
linha1 = raw_input()
A,B,C,D = linha1.split(" ")
A = float(A)*2
B = float(B)*3
C = float(C)*4
D = float(D)*1
Media = (A + B + C+D) / 10
print("Media: %.1f" %Media)
if Media>=7:
    print("Aluno aprovado.")
elif Media<5:
    print("Aluno reprovado.")
elif Media>=5 and Media<=6.9:
    print("Aluno em exame.")
    exame = float(input())
    print("Nota do exame: %.1f" %exame)
    mediadepoisexame = (Media+exame)/2
    if mediadepoisexame>=5:
        print("Aluno aprovado.")
    else:
        print("Aluno aprovado.")
    print("Media final: %.1f" %mediadepoisexame)    
        

File: C/A_Nivel_Iniciante/Exercicio_1040.cpp
# include <stdio.h>
int main(){
	float n1,n2,n3,n4,nota_exame,media_final;
	scanf("%f %f %f %f",&n1,&n2,&n3,&n4);
	n1*=2;
	n2*=3;
	n3*=4;
	n4*=1;
	float media = (n1 + n2 + n3 + n4)/10.0;
	printf("Media: %.1f\n",media);
    if(media>=7){
		printf("Aluno aprovado.\n");
	}else if(media<5){
		printf("Aluno reprovado.\n");
	} 
    if(media>=5 && media<7){
    	printf("Aluno em exame.\n");
    	scanf("%f",&nota_exame);
    	printf("Nota do exame: %.1f\n",nota_exame);
    	media_final = (nota_exame + media)/2;
    	if(media>=5){
		printf("Aluno aprovado.\n");
		}else if(media<5){
			printf("Aluno reprovado.\n");
		}	 
    	printf("Media final: %.1f\n",media_final);
	}
	return 0;
}
